expan/core/early_stopping.py

```diff
@@ -56,7 +59,9 @@ def group_sequential(x,
             the end of experiment
         alpha: type-I error rate
         cap: upper bound of the adapted z-score
-
+        multi_test_correction: multiple correction flag
```

Comment:
how about "flag of whether the correction for multiple testing is needed" to be consistent?

Reply:
could more clear, thanks!